validate.go

```diff
@@ -44,6 +44,12 @@ func IsGoodHash(code uint64) bool {
 			return true
 		}
 	}
+	// XXX: This has to be on par with what the LazyLedger IPLD plugin registers as a multihash
+	// namely, Sha256Namespace8Flagged. We simply repeat the constant here instead of
+	// importing the corresponding package from lazyledger-core as this would be overkill.
+	if code == 0x7701 {
+		return true
+	}
 
 	return false
```

Comment:
This can be simplified to `return code == 0x7701`.

Reply:
I'll update that, merge the PR and bump the deps in the corresponding go.mod files ...